test.tf

```diff
@@ -35,6 +35,8 @@ resource "aws_ami" "awsAmiEncrypted" {
     device_name = "dev-name"
     encrypted = "false"
   }
+  
+  
 }
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure at-rest server side encryption (SSE) is enabled using AWS KMS for AWS S3 Buckets**
 
**Vulnerability**
AWS S3 Buckets do not have at-rest server side encryption enabled to protect sensitive data.
 
**Remediation**
Server-side encryption protects data at rest. Amazon S3 encrypts each object with a unique key. As an additional safeguard, it encrypts the key itself with a key that it rotates regularly. Amazon S3 server-side encryption uses one of the strongest block ciphers available to encrypt your data using AWS KMS Customer managed key.
 In Terraform - 
 Add the 'server_side_encryption_configuration' block to ensure server side encryption is enabled. Ensure 'see_algorithm' is set to 'aws:kms' and add a 'kms_master_key_id'.

source: tenablecs-